spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/core/SpannerObjectMapperImplTests.java

```diff
@@ -356,6 +358,34 @@ public void writeIncompatibleTypeTest() {
 		ByteArray bytes;
 	}
 
+	@SpannerTable(name = "custom_test_table_reduced")
+	private static class TestEntityReduced {
```

Comment:
why is it reduced? compared to what?

Reply:
Well previously it was ok that the entity had many more properties than there were columns, but now the opposite is true. to avoid writing out all those extra lists ( boolean, date, timestamp, long , string, etc...) I cut those out and made this new entity type.

Reply:
Hmm...but then we'll have asymmetry between the read and write side, won't we? we're not covering reading those type of lists anywhere else, or do we? Is there anything bad about writing out those lists?

Reply:
no technically it is not worse to write out those other cases, but I felt it would be obtuse since we're just getting these mapping methods from a Map we hard coded.

Reply:
I think it's valuable to cover our map, and that avoids having to create a separate class just to avoid writing more too.